doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -26,23 +26,23 @@ Optionally, you can set custom configuration for {KIE_SERVERS}. In this case, de
 +
-. If you want to configure an immutable {KIE_SERVER}, complete the following additional steps:
+. If you want to configure an immutable {KIE_SERVER} using a Source to Image (S2I) build, complete the following additional steps:
++
+[IMPORTANT]
+====
+If you want to configure an immutable {KIE_SERVER} that pulls services from the Maven repository, do not click the *Set Immutable server configuration* button and do not complete these steps. Instead, set the `KIE_SERVER_CONTAINER_REPLOYMENT` environment variable.
+====
++
 .. Click *Set Immutable server configuration*.
-.. In the *KIE Server container deployment* field, enter the identifying information of the services (KJAR files) that the deployment must pull from the Maven repository or from the result of a Source to Image (S2I) build. The format is `<containerId>=<groupId>:<artifactId>:<version>` or, if you want to specify an alias name for the container, `<containerId>(<aliasId>)=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, as illustrated in the following example: `containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2`.
+.. In the *KIE Server container deployment* field, enter the identifying information of the services (KJAR files) that the deployment must take from the result of a Source to Image (S2I) build. The format is `<containerId>=<groupId>:<artifactId>:<version>` or, if you want to specify an alias name for the container, `<containerId>(<aliasId>)=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, as illustrated in the following example: `containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2`.
 .. If your OpenShift environment does not have a connection to the public Internet, enter the URL of the Maven mirror that you set up according to <<offline-repo-proc_{context}>> in the *Maven mirror URL* field.
 .. If you want to use an S2I build, the *Artifact directory* field, enter the path within the project that contains the required binary files (KJAR files and any other necessary files) after a successful Maven build. Normally this directory is the target directory of the build. However, you can provide prebuilt binaries in this directory in the Git repository.
 .. If you want to use a custom base {KIE_SERVER} image for the S2I build, click *Set Base build image* and then enter the name of the image stream in the *Name* field. If the image stream is not in the `openshift` namespace, enter the namespace in the *Namespace* field. If you want to use a Docker image name and not an OpenShift image stream tag, change the *Kind* value to `DockerImage`.
-.. If you want to create the immutable server from source in a Git repository using the Source to Image (S2I) process, click the *Set Git source* button and enter information in the following fields:
+.. Click the *Set Git source* button and enter information in the following fields:
```

Comment:
Suggestion - adopt UI style for buttons from IBMSG 212p:

`s/Click the *Set Git Source* button and/Click *Set Git Source* and`